app/javascript/controllers/start_tournament_controller.js

```diff
@@ -5,7 +5,7 @@ export default class extends Controller {
   static targets = [ "form", "modal", "close" ];
 
   connect() {
-    console.log("connecteddd");
+    // console.log("connecteddd");
```

Comment:
why is it connecteDDD? is it because you have many connect?

Reply:
I did this once upon a time, cause sometimes I do connected and if you do the same word every time it's not always clear which controller youre connected from